site2/docs/admin-api-brokers.md

```diff
@@ -117,15 +129,20 @@ It finds all namespaces which are owned and served by a given broker.
 
 ```shell
 pulsar-admin brokers namespaces use \
---url broker1.use.org.com:8080
+    --url localhost:8080
+```
 
-{
-  "my-property/use/my-ns/0x00000000_0xffffffff": {
-    "broker_assignment": "shared",
-    "is_controlled": false,
-    "is_active": true
-  }
-}
+Example output:
+
+```shell
```

Comment:
````suggestion
```
````

Output should not has a highlight tag.

Reply:
ditto others. If the output is in JSON format, you can use `json`.

Reply:
Make sense, I'll fix it.